src/test/java/com/sugarcrm/candybean/automation/webdriver/WebDriverElementSystemTest.java

```diff
@@ -343,6 +374,58 @@ public void getCssValueTest() throws CandybeanException {
 	}
 
 	@Test
+	public void waitForAttribute() throws Exception {
+		iface.go("file://"+ System.getProperty("user.dir")+"/resources/html/test/blinking.html");
+		WebDriverElement text = iface.getWebDriverElement(new Hook(Strategy.ID, "p1"));
+		text.click();
+		Assert.assertTrue(iface.getPause().waitForAttribute(new Hook(Strategy.ID, "p1"), "class", "hidden", true, 10) != null);
+		Assert.assertTrue(iface.getPause().waitForAttribute(new Hook(Strategy.ID, "p1"), "class", "hidden", false, 10) != null);
+		Assert.assertTrue(iface.getPause().waitForAttribute(new Hook(Strategy.ID, "p1"), "class", "normal", false, 10) != null);
+		Assert.assertTrue(iface.getPause().waitForAttribute(new Hook(Strategy.ID, "p1"), "class", "normal", false, 10) != null);
```

Comment:
Two lines in a row look identical -- is this intentional, or should one be passing true?